src/components/Popup.jsx

```diff
@@ -7,25 +7,25 @@ class Popup extends React.Component {
     this.popupBody = null;
     this.close = this.close.bind(this);
   }
-  
+
   render() {
     return (
       <div className="popup" ref={(c)=>{this.popupBody=c}} onClick={(e) => { return e.target === this.popupBody && this.close(e); }}>
         <div className="popup-title">
-          <button className="fa fa-close" onClick={this.close}></button>
+          <i className="fa fa-close" onClick={this.close}></i>
```

Comment:
Query: why are we replacing `<button>` with `<i>` ? With `<button>`, users can use Tab navigation to reach the close button; while `<i>` reduces keyboard accessibility.

However, I can understand this change if the goal is to later add a "Press `Esc` to close popup" feature, which may actually be the best way forward. (Our current keyboard accessibility is sub-optimal, to say the least.)